### PR TITLE
WFNEWS-1451 Full details response tab

### PR DESCRIPTION
**Pull request by ssylver93**

### CODE REVIEW

File: client/wfnews-war/src/main/angular/src/app/components/admin-incident-form/admin-incident-form.component.ts [hunk 1]
```diff
@@ -348,6 +348,8 @@ export class AdminIncidentForm implements OnInit, OnChanges {
     };
 
     try {
+      console.log('jsonn')
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/admin-incident-form/admin-incident-form.component.ts:351]: You can remove console logs once you're done with them

File: client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-overview-panel-mobile/incident-overview-panel-mobile.component.html [hunk 2]
```diff
@@ -1,11 +1,62 @@
...
+            <div class="resource-panel" *ngIf="incident.incidentMgmtCrewRsrcInd">
+                <div class="subtitle"><img class="image-size"
+                        src="../../../assets/images/svg-icons/incident_management_team.svg"
+                        alt="Incident Management Team" />Incident Management Team</div>
+                <div class="contents">{{incident.incidentMgmtCrewRsrcDetail}}</div>
+                <hr>
+            </div>
+            <div class="resource-panel" *ngIf="incident.wildfireCrewResourcesInd">
+                <div class="subtitle"><img class="image-size"
+                        src="../../../assets/images/svg-icons/firefighting_personnel.svg"
```
Comment (dhlevi) [inline client/wfnews-war/src/main/angular/src/app/components/public-incident-page/incident-overview-panel-mobile/incident-overview-panel-mobile.component.html:24]: You should be able to set the src paths to just `/assets/images/...`. Prevents possible path issues

**Review by dhlevi — APPROVED** [review]
Some minor issues to clean up but otherwise looks like a good start.

File: client/wfnews-war/src/main/angular/src/app/components/admin-incident-form/admin-incident-form.component.ts (hunk 1 above)
Comment (ssylver93) [reply client/wfnews-war/src/main/angular/src/app/components/admin-incident-form/admin-incident-form.component.ts]: Done, thanks

**Pull request merged**